src/views/email-2022.js

```diff
@@ -0,0 +1,238 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AppConstants from '../app-constants.js'
+import { getMessage } from '../utils/fluent.js'
+
+const companyAddress = '2 Harrison St. #175, San Francisco, California 94105 USA'
+const links = (data) => ({
+  faq: 'https://support.mozilla.org/kb/firefox-monitor-faq',
+  hibp: 'https://haveibeenpwned.com/',
+  legal: `https://www.mozilla.org/about/legal?utm_source=fx-monitor&utm_medium=email&utm_campaign=${data.utmCampaign}&utm_content=email-footer-link`,
+  termsAndPrivacy: `https://www.mozilla.org/privacy/firefox-monitor?utm_source=fx-monitor&utm_medium=email&utm_campaign=${data.utmCampaign}&utm_content=email-footer-link`
+})
+
+const images = {
+  header: `${AppConstants.SERVER_URL}/img/email_images/person-at-desk.png`,
+  footer: `${AppConstants.SERVER_URL}/img/email_images/mozilla-logo-bw-rgb.png`,
+  logoDark: `${AppConstants.SERVER_URL}/img/email_images/monitor-logo-transparent-dark-mode.png`,
+  logoLight: `${AppConstants.SERVER_URL}/img/email_images/monitor-logo-bg-f9f9fa.png`
+}
```

Comment:
Woops, almost missed this.  I think these paths refer to the old site.  We need to duplicate these images into the `src/client/images` folder.  And another twist – our new repo is using .webp and we should avoid .png (the build phase is not set to handle .png files)

I believe @flozia knows how to use the `convert:webp` script in `src/package.json`, and perhaps this is all already handled in the frontend PR for this feature?  I'll take a look next.

Reply:
@toufali That’s my bad, I only learned about our use of `webp` after introducing this part of the email and did not update respectively. @rhelmer Opened PR #2799 which adds the converted images.

Reply:
Thanks for catching this! @flozia took care of this for us in https://github.com/mozilla/blurts-server/pull/2799